sensors:

     ultrasound_rr_sag:
        layer: rr_sag
        topic_name: /bosch_uss5/ultrasound_rr_sag
        detection_range: 0.4

     ultrasound_rr_lat:
        layer: rr_lat
        topic_name: /bosch_uss5/ultrasound_rr_lat
        detection_range: 0.4

     ultrasound_fr_sag:
        layer: fr_sag
        topic_name: /bosch_uss5/ultrasound_fr_sag
        detection_range: 0.4

     ultrasound_fr_lat:
        layer: fr_lat
        topic_name: /bosch_uss5/ultrasound_fr_lat
        detection_range: 0.4

     ultrasound_fl_sag:
        layer: fl_sag
        topic_name: /bosch_uss5/ultrasound_fl_sag
        detection_range: 0.4

     ultrasound_fl_lat:
        layer: fl_lat
        topic_name: /bosch_uss5/ultrasound_fl_lat
        detection_range: 0.4

     ultrasound_rl_sag:
        layer: rl_sag
        topic_name: /bosch_uss5/ultrasound_rl_sag
        detection_range: 0.4

     ultrasound_rl_lat:
        layer: rl_lat
        topic_name: /bosch_uss5/ultrasound_rl_lat
        detection_range: 0.4



